types/MomentLocaleUtils.d.ts

```diff
@@ -1,2 +1,2 @@
-import { LocaleUtils } from './utils';
+import { LocaleUtils } from './LocaleUtils';
 export const MomentLocaleUtils: LocaleUtils;
```

Comment:
@gpbl 
This type isn't correct, when trying to import
```ts
import utils from 'react-day-picker/moment';
```

The resulting type for `utils` is `{ MomentLocaleUtils: LocaleUtils }`.
I think the proper fix would be:

```ts
export default LocaleUtils;
```

Reply:
Hey thanks for the heads up! releasing a patch...

Reply:
Should be fixed in v7.4.7.

Reply:
Thanks! It's working great.

Reply:
phew!!